tfhe/src/core_crypto/entities/seeded_lwe_compact_public_key.rs

```diff
@@ -229,3 +230,18 @@ impl<Scalar: UnsignedInteger> SeededLweCompactPublicKeyOwned<Scalar> {
         )
     }
 }
+
+impl<C: Container> ParameterSetConformant for SeededLweCompactPublicKey<C>
+where
+    C::Element: UnsignedInteger,
+{
+    type ParameterSet = GlweCiphertextConformanceParameters<C::Element>;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool {
+        let Self {
+            seeded_glwe_ciphertext,
+        } = self;
+
```

Comment:
same here we need to check that the lwe dim is a power of 2 and glwe dim = 1 before chekcing the glwe